src/Constants/Plugin.php

```diff
@@ -46,10 +46,13 @@ class Plugin
     const CONFIG_FILTERS_STYLING_CSS_ENABLED = 'Findologic.filters_styling_css_enabled';
     const CONFIG_LOAD_NO_UI_SLIDER_STYLES_ENABLED = 'Findologic.load_no_ui_slider_styles_enabled';
     const CONFIG_MINIMAL_SEARCH_TERM_LENGTH = 'Findologic.minimalSearchTermLength';
-
+    const CONFIG_USE_VARIANTS = 'Findologic.useVariants';
     const CONFIG_IO_CATEGORY_SEARCH = 'IO.routing.category_search';
 
     const FILTER_TYPE_RANGE_SLIDER = 'range-slider';
     const FILTER_TYPE_COLOR = 'color';
     const FILTER_TYPE_IMAGE = 'image';
+
+    const FILTER_HANDLER_DELIMITER = '|';
+    const FILTER_VALUE_DELIMITER = '>';
```

Comment:
Both should be unused, as we are not using them like in Shopware 6. Within plenty we use the default structure `&attrib[vendor][0]=Exclusive Leather&attrib[vendor][1]=Marc O'Polo`, but Shopware does that with separators.